teuthology/kill.py

```diff
@@ -84,6 +86,10 @@ def kill_job(run_name, job_id, archive_base=None, owner=None, skip_nuke=False):
                 "Please pass --owner <owner>.")
         owner = job_info['owner']
     kill_processes(run_name, [job_info.get('pid')])
+    if 'machine_type' in job_info:
+        teuthology.exporter.JobResults().record(job_info["machine_type"], job_info["status"])
```

Comment:
what if  ``job_info["status"]`` is None, wanna also add `` and job_info.get('status')`` to the if statement?

Reply:
hm actually I think we might want to always use the literal `'dead'` here, since we've just killed the process itself!